Core/FileSystems/BlockDevices.cpp

```diff
@@ -197,7 +197,8 @@ CISOFileBlockDevice::CISOFileBlockDevice(FileLoader *fileLoader)
 	u64 lastIndexPos = index[indexSize - 1] & 0x7FFFFFFF;
 	u64 expectedFileSize = lastIndexPos << indexShift;
 	if (expectedFileSize > fileSize) {
-		ERROR_LOG(LOADER, "Expected CSO to at least be %lld bytes, but file is %lld bytes", expectedFileSize, fileSize);
```

Comment:
Hm, I guess it could be `((lastIndexPos - 1) << indexShift) + 1` and still be valid, but otherwise the file is definitely corrupt.  Is this common enough not to consider it an error?

-[Unknown]

Reply:
I kind of feel like ERROR is more for when something has definitely gone wrong, while WARN is more appropriate for user data (like ISOs) being broken. What an error is is kind of a philosophical question. I don't mind changing it back to ERROR though.

(I got this warning on my mac because I had some broken file sitting around. But arguably we should mark it as broken in the UI too, and so on, which would be a bigger change...)